tutorials-v5/time-evolution/013_nonmarkovian_monte_carlo.md

```diff
@@ -5,13 +5,27 @@ jupyter:
       extension: .md
       format_name: markdown
       format_version: '1.3'
-      jupytext_version: 1.15.2
+      jupytext_version: 1.16.1
   kernelspec:
     display_name: Python 3 (ipykernel)
     language: python
     name: python3
 ---
 
+```python
+import matplotlib.pyplot as plt
+from matplotlib.collections import LineCollection
+from matplotlib.colors import LogNorm
+
+import numpy as np
+import qutip as qt
+
+from scipy import special, optimize
+from scipy.interpolate import CubicSpline
+
+import os
```

Comment:
Why did the imports move up to the top here?

Reply:
The notebook contains two examples. These are the imports for both examples, so it seemed more logical to have them before the start of the first example. If one wants, they can execute the imports, skip the first example and continue with the second.
I'm not sure what the best practice is, we could also split it into two import statements, one at the beginning of each example.

Reply:
I think one block is fine, but could we move it after the introduction so that the heading and description of the notebook come before it like in our other notebooks?

Let's also add something to the description that mentions the MPI example at the end.